util/conan-package/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake
+import os
+
+channel = os.getenv("CONAN_CHANNEL", "testing")
```

Comment:
From conan 0.25 you can simplify this test_package recipe, removing the user/channel, and also the ``requires``, so it is not necessary to change them or change versions when testing a different one. Then, use ``conan create user/channel`` better.